module/resource-tracker/index.ts

```diff
@@ -0,0 +1,62 @@
+import { ResourceTrackerManager } from './resource-tracker-manager.js'
+import { OLD_SETTING_TEMPLATES, SETTING_TRACKER_EDITOR, SETTING_TRACKER_TEMPLATES } from './types.js'
+
+export function init() {
+  console.log('GURPS | Initializing GURPS Resource Tracker Module')
+  Hooks.once('ready', async function () {
+    if (!game.settings) throw new Error('GURPS | Game settings not found')
+    if (!game.i18n) throw new Error('GURPS | Game i18n not found')
+
+    // TODO: Remove this when the setting is removed.
+    game.settings.register(GURPS.SYSTEM_NAME, OLD_SETTING_TEMPLATES, {
```

Comment:
I've been considering moving `game.settings.register` calls for modules to a `<module name>/settings.ts` file rather than having them in the index, just to make the index cleaner. What do you think?

Reply:
For many modules, this is the only setup required, so I'm good with leaving them in the init() function.